client/src/app/components/footer/footer.component.html

```diff
@@ -1,10 +1,50 @@
-<div class="text-center fixed-bottom bg-light">
-  <span class="mx-1">
-      <i class="fa fa-facebook cursor-click fa__icon__black"></i>
-  </span>
-  <span class="mx-2">
-      <i class="fa fa-linkedin cursor-click fa__icon__black"></i>
-  </span>
-
-  <small>©2018 · All rights reserved</small>
+<div class="footer text-center fixed-bottom bg-light">
```

Comment:
Finally something easy to read, straightforward classes, clean structuring of "levels"... 👍